Applications/ConsumerApi/test/ConsumerApi.Tests.Integration/StepDefinitions/RelationshipsStepDefinitions.cs

```diff
@@ -65,6 +70,26 @@ public async Task GivenAPendingRelationshipBetweenI1AndI2CreatedByI1()
         _relationshipId = createRelationshipResponse.Result!.Id;
     }
 
+    [Given("an active Relationship between i1 and i2 created by i1")]
+    public async Task GivenAnActiveRelationshipBetweenI1AndI2()
+    {
+        var relationshipTemplateResponse = await CreateRelationshipTemplate(_client1);
+        var createRelationshipResponse = await CreateRelationship(_client2, relationshipTemplateResponse.Result!.Id);
+        var acceptRelationshipResponse = await AcceptRelationship(_client1, createRelationshipResponse.Result!.Id);
```

Comment:
There's a method in the `Utils.cs` class for establishing a relationship you can use `EstablishRelationshipBetween`.
Maybe even build on top of that method to establish Relationships in a given status (e.g., Accepted, Reject, etc.)

Reply:
done

Reply:
The step definition for `GivenARejectedRelationshipBetweenI1AndI2` could also benefit from this

Reply:
So should I just create a new method like `CreateRejectedRelationshipBetween` and then use it?

Reply:
Either that or change the current one to something like `CreateRelationshipInStatus` and add support for various ones you need.

Reply:
done